mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_holdings_and_srs_marc_holdings_create_job_profile.sql

```diff
@@ -0,0 +1,299 @@
+INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
+('fa0262c7-5816-48d0-b9b3-7b7a862a5bc7', '{
+  "id": "fa0262c7-5816-48d0-b9b3-7b7a862a5bc7",
+  "name": "quickMARC Derive - Create Holdings and SRS MARC Holdings",
+  "description": "Load MARC Holdings and inventory holdings to create SRS MARC Holdings and Holdings",
+  "deleted": false,
+  "dataType": "MARC",
+  "tags": {
+    "tagList": []
+  },
+  "childProfiles": [],
+  "parentProfiles": [],
+  "userInfo": {
+    "firstName": "System",
+    "lastName": "System",
+    "userName": "System"
+  },
+  "metadata": {
+    "createdDate": "2021-03-16T15:00:00.000",
+    "updatedDate": "2021-03-16T15:00:00.000",
+    "createdByUserId": "00000000-0000-0000-0000-000000000000",
+    "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+  }
+}')
+ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.action_profiles (id, jsonb) values
+('f5feddba-f892-4fad-b702-e4e77f04f9a3', '{
+  "id": "f5feddba-f892-4fad-b702-e4e77f04f9a3",
+  "name": "Default derive - Create holdings",
+  "action": "CREATE",
+  "deleted": false,
+  "description": "This action profile is used with FOLIO''s default job profile for creating Inventory Holdings and SRS MARC Holdings records. It can be edited, duplicated.",
+  "folioRecord": "HOLDINGS",
+  "childProfiles": [],
+  "parentProfiles": [],
+    "metadata": {
+      "createdDate": "2021-08-05T14:00:00.000",
+      "updatedDate": "2021-08-05T15:00:00.462+0000",
+      "createdByUserId": "00000000-0000-0000-0000-000000000000",
+      "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+    },
+    "userInfo": {
+      "lastName": "System",
+      "userName": "System",
+      "firstName": "System"
+    }
+}')
+ON CONFLICT DO NOTHING;
+
+
+INSERT INTO ${myuniversity}_${mymodule}.mapping_profiles (id, jsonb) values
+('e0fbaad5-10c0-40d5-9228-498b351dbbaa', '{
+  "id": "e0fbaad5-10c0-40d5-9228-498b351dbbaa",
+  "name": "Default derive - Create holdings",
+  "deleted": false,
+  "metadata": {
+    "createdDate": "2021-08-05T14:00:00.000",
+    "updatedDate": "2021-08-05T15:00:00.462+0000",
+    "createdByUserId": "00000000-0000-0000-0000-000000000000",
+    "updatedByUserId": "00000000-0000-0000-0000-000000000000"
+  },
+  "userInfo": {
+    "lastName": "System",
+    "userName": "System",
+    "firstName": "System"
+  },
+  "description": "This field mapping profile is used with FOLIO''s default job profile for creating Inventory Holdings and SRS MARC Holdings records. It can be edited, duplicated, deleted, or linked to additional action profiles.",
```

Comment:
update the description, please - "It cannot be edited or deleted." - https://github.com/folio-org/mod-data-import-converter-storage/blob/master/mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_instance_and_srs_marc_bib_create_job_profile.sql#L66